Core/MIPS/ARM64/Arm64Asm.cpp

```diff
@@ -315,6 +316,8 @@ void Arm64Jit::GenerateFixedCode(const JitOptions &jo) {
 
 	// Don't forget to zap the instruction cache! This must stay at the end of this function.
 	FlushIcache();
+
+	js.currentRoundingFunc = convertS0ToSCRATCH1[0];
```

Comment:
Not sure if this is safe.  `ClearCache()` will call this function and should not reset `js.currentRoundingFunc`.

It makes more sense to do this in the constructor after `js.startDefaultPrefix = mips_->HasDefaultPrefix();`.

-[Unknown]